docs/docs/building-apps-with-gatsby.md

```diff
@@ -2,15 +2,18 @@
 title: "Building Apps with Gatsby"
 ---
 
-Gatsby is an excellent framework for building web apps. You can use Gatsby to create personalized, logged-in experiences with two different methods.
+Gatsby is an excellent framework for building web apps. You can use Gatsby to create personalized, logged-in experiences with two different approaches.
 
-The first approach is to build "hybrid" app pages which are statically rendered with dynamic sections. The second is, if needed, add client-only multi-page sections of the site.
+1. "hybrid" app pages, and
+2. client-only routes & user authentication
 
 ## Hybrid app pages
 
-With this method, Gatsby renders the initial page with shared page content -- then when your React components load in the browser, they can fetch and render data from APIs. The [React docs have a simple example of how to do this.](https://reactjs.org/docs/faq-ajax.html)
+With this method, Gatsby initially renders the page statically, and then when your React components load in the browser they can fetch and render data from APIs.
```

Comment:
This is kinda awkward language — perhaps change it to describe how a visitor to a Gatsby site first loads an HTML file and then the JavaScript.

Reply:
Modified the language